packages/crowdsourcing/graphql/resolvers/_mutations/submitInheritanceStory.js

```diff
@@ -54,9 +54,15 @@ ${content ? 'Erbgeschichte: ' + content : ''}
 
 `
 
-  return sendMail({
-    to: CS_INHERITANCE_TO,
-    subject: CS_INHERITANCE_SUBJECT,
-    text: emailText
-  }, context)
+  return sendMail(
+    {
+      to: CS_INHERITANCE_TO,
+      subject: CS_INHERITANCE_SUBJECT,
+      text: emailText
+    },
+    context,
+    {
+      logMessage: false
```

Comment:
```suggestion
      logMail: false
```
After thinking about it `logMail` feels better than `logMessage`. An email is an email and less a message. Or just shorten it to `log: false`

Reply:
It still logs whole "email" just not it's content. And since content is wrapped in `message`, would like to stick to it.